utils/funcexp/func_ltrim_oracle.cpp

```diff
@@ -91,7 +89,7 @@ std::string Func_ltrim_oracle::getStrVal(rowgroup::Row& row, FunctionParm& fp, b
   std::string ret(pos, binLen);
   if (binLen == 0)
   {
-    isNull = true;
+    isNull = true; // XXX: is it goog?
```

Comment:
Good question. Did you try this function?

Reply:
We have no tests for that function in our MTR.

https://mariadb.com/docs/skysql/ref/mdb/functions/LTRIM_ORACLE/

"This accesses a version of the LTRIM function that returns NULL instead of an empty string when returning an empty string."

Wowsers. ;)

Reply:
I'll look into that.

Reply:
ltrim_oracle does not work as intended.

Reply:
Fixed, added to the appropriate test.